# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 7ecc4f9b20ad
# Full Hash: 7ecc4f9b20ad70f95a63b96d8a53ebb36ee32d99
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 3
# Description:
#   Bug 1742437 - move DOM script context into own file and rename it; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138132
# ==============================================================================

diff -r 0b3f0972e002 -r 7ecc4f9b20ad dom/script/moz.build
--- a/dom/script/moz.build	Mon Feb 21 15:39:08 2022 +0000
+++ b/dom/script/moz.build	Mon Feb 21 15:39:08 2022 +0000
@@ -26,6 +26,7 @@
     "ScriptDecoding.h",
     "ScriptElement.h",
     "ScriptKind.h",
+    "ScriptLoadContext.h",
     "ScriptLoader.h",
     "ScriptLoadRequest.h",
     "ScriptSettings.h",
@@ -40,6 +41,7 @@
     "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
     "ScriptElement.cpp",
+    "ScriptLoadContext.cpp",
     "ScriptLoader.cpp",
     "ScriptLoadHandler.cpp",
     "ScriptLoadRequest.cpp",
